serverless/DEVELOPER_README.md

```diff
@@ -61,15 +61,15 @@ The `DefaultDatabase`, `DefaultTable` and `LogLevel` may be altered to fit your
-| Europe (Ireland) eu-west-1 | [View](https://timestreamassets-eu-west-1.s3.amazonaws.com/timestream-prometheus-connector/template.yml) |  [View in Composer](https://console.aws.amazon.com/composer/canvas?region=eu-west-1&templateURL=https://timestreamassets-eu-west-1.s3.amazonaws.com/timestream-prometheus-connector/template.yml)           | [Launch](https://console.aws.amazon.com/cloudformation/home?region=eu-west-1#/stacks/new?stackName=PrometheusTimestreamConnector&templateURL=https://timestreamassets-eu-west-1.s3.amazonaws.com/timestream-prometheus-connector/template.yml)         |
+| Region                    | View                                                                                                                                      | View in Composer                                                                                                                                                                                                                                         | Launch                                                                                                                                                                                                                                                                            | Launch (SigV4) |
+|---------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|
+| US East (N. Virginia) us-east-1 | [View](https://timestreamassets-us-east-1.s3.amazonaws.com/timestream-prometheus-connector/template.yml) |  [View in Composer](https://console.aws.amazon.com/composer/canvas?region=us-east-1&templateURL=https://timestreamassets-us-east-1.s3.amazonaws.com/timestream-prometheus-connector/template.yml)           | [Launch](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=PrometheusTimestreamConnector&templateURL=https://timestreamassets-us-east-1.s3.amazonaws.com/timestream-prometheus-connector/template.yml)         | [Launch (SigV4)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/create/review?stackName=PrometheusTimestreamConnector&templateURL=https://timestreamassets-us-east-1.s3.amazonaws.com/timestream-prometheus-connector/template.yml&param_EnableSigV4Auth=true) |
```

Comment:
This hasn't exactly been tested. This requires that the template be updated in all of our buckets. I have tested this method for setting parameters with one-click deployment, but with other parameters that already exist in the currently uploaded templates.

Note the change from the "Launch" link's `stacks/new?` to `stacks/create/review?`. This is required in order to set parameters in URLs.